Kernel/Memory/MemoryManager.cpp

```diff
@@ -427,6 +427,12 @@ UNMAP_AFTER_INIT void MemoryManager::initialize_physical_pages()
                 highest_physical_address = range_end;
         }
 
+        if ((multiboot_flags & MULTIBOOT_INFO_FRAMEBUFFER_INFO) && !multiboot_framebuffer_addr.is_null() && multiboot_framebuffer_type == MULTIBOOT_FRAMEBUFFER_TYPE_RGB) {
```

Comment:
Is this the only way to actually consider the framebuffer region? a memory map to see from that machine will be useful.

Reply:
You can also look at PCI BAR registers. It's not in a memory map. But it's in a PCI bars. GFX has BARs 0x90000000+16M and 0x80000000+256M. Later one is framebuffer.